src/main/java/core/basesyntax/dao/writeReport.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.dao;
+
+public interface writeReport {
```

Comment:
The interface name `writeReport` does not follow Java naming conventions. Interface names should be in PascalCase, such as `WriteReport`.